oeps/architectural-decisions/oep-0030-arch-pii-markup-and-auditing.rst

```diff
@@ -307,13 +287,13 @@ Backward Compatibility
 **********************
 The proposed updates do not introduce any known backward incompatibilities, but would require a comprehensive effort to annotate existing PII in all Open edX repositories. The desire for that effort is what drove the initial tasks that led to this OEP, so this is not undesirable or duplicate work.
 
+As of 2025 edx-platform and all of its dependencies are annotated to 88% coverage, with the remaining 12% being in the process of being annotated. The enforcement tooling is in place and is blocking new PII from being added to the platform without annotation.
```

Comment:
```suggestion
As of February 2025 edx-platform and all of its dependencies are annotated to 88% coverage, with the remaining 12% being in the process of being annotated. The enforcement tooling is in place and is blocking new PII from being added to the platform without annotation.
```

Mostly because 2025 is long and this is really a point-in-time comment.